### PR TITLE
Basic Sass tests for `$theme-colors` customization

### PR DESCRIPTION
**Pull request by mdo**
This is a super simple Sass test to demonstrate if our maps are working as intended.

```scss
@import "../scss/functions";

$custom-color: #df711b;
$custom-theme-colors: (
  "custom": $custom-color
);

@import "../scss/variables";

$theme-colors: map-merge($theme-colors, $custom-theme-colors);

@import "../scss/maps";

@debug "The following three maps should be the same, ending with the custom key.";
@debug map.keys($theme-colors);
@debug map.keys($theme-colors-rgb);
@debug map.keys($utilities-colors);
@debug "";
@debug "The following two maps should be the same: including custom and ending in black, white, and body.";
@debug map.keys($utilities-text-colors);
@debug map.keys($utilities-bg-colors);
```

```sh
➜ ~/work/bootstrap (split-vars-maps-import) npm run css-test

> bootstrap@5.1.1 css-test /Users/mdo/work/bootstrap
> sass --style expanded --source-map --embed-sources --no-error-css test/index.scss:test/output.css

test/index.scss:35 Debug: The following three maps should be the same, ending with the custom key.
test/index.scss:36 Debug: "primary", "secondary", "success", "info", "warning", "danger", "light", "dark", "custom"
test/index.scss:37 Debug: "primary", "secondary", "success", "info", "warning", "danger", "light", "dark", "custom"
test/index.scss:38 Debug: "primary", "secondary", "success", "info", "warning", "danger", "light", "dark", "custom"
test/index.scss:40 Debug: 
test/index.scss:41 Debug: The following two maps should be the same: including custom and ending in black, white, and body.
test/index.scss:42 Debug: "primary", "secondary", "success", "info", "warning", "danger", "light", "dark", "custom", "black", "white", "body"
test/index.scss:43 Debug: "primary", "secondary", "success", "info", "warning", "danger", "light", "dark", "custom", "black", "white", "body"
```

### CODE REVIEW

**Comment by ffoodd** [comment]
Would there be a way to output boolean values, and maybe handle exit codes for proper CI & test integration?

Can't play around this for now but even if this is a great step already, it requires we read the output, doesn't it?

**Comment by mdo** [comment]
This is definitely a spike I want to revisit with `@error` instead of just `@debug`. For example, we could write something like:

```scss
@if map.get($theme-colors, "primary") != blue {
  @error "That's not the right color.";
}
```

Which would output:

```sh
Error: "That's not the right color."
   ╷
29 │   @error "That's not the right color.";
   │   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
  test/index.scss 29:3  root stylesheet
```

**Comment by mdo** [comment]
Latest commits include new tests @ffoodd that use `@warn` to pass output to the CLI. This should then be trigger-able in Actions to cause a failing test. `@error` isn't being used because it stops the CSS from being compiled _at that error_, so it wouldn't show all the errors in the entire test file.

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "css-prefix-main": "postcss --config build/postcss.config.js --replace \"dist/css/*.css\" \"!dist/css/*.rtl*.css\" \"!dist/css/*.min.css\"",
     "css-prefix-examples": "postcss --config build/postcss.config.js --replace \"site/content/**/*.css\"",
     "css-prefix-examples-rtl": "cross-env-shell NODE_ENV=RTL postcss --config build/postcss.config.js --dir \"site/content/docs/$npm_package_config_version_short/examples/\" --ext \".rtl.css\" --base \"site/content/docs/$npm_package_config_version_short/examples/\" \"site/content/docs/$npm_package_config_version_short/examples/{blog,carousel,dashboard,cheatsheet}/*.css\" \"!site/content/docs/$npm_package_config_version_short/examples/{blog,carousel,dashboard,cheatsheet}/*.rtl.css\"",
+    "css-test": "sass --style expanded --source-map --embed-sources --no-error-css test/index.scss:test/output.css",
```
Comment (XhmikosR) [inline package.json:38]: Shouldn't we gitignore the 2 new output.css and output.map files?

**Pull request closed without merging**